### ISSUE TITLE
Document Loaders: GoogleDriveLoader can't load a google docs file.

### ISSUE THREAD

**Issue by sopmacvan** [langchain-ai/langchain#2579]
Following this [guide](https://python.langchain.com/en/latest/modules/indexes/document_loaders/examples/googledrive.html), I tried to load my google docs file [doggo wikipedia](https://docs.google.com/document/d/1SJhVh8rQE7gZN_iUnmHC9XF3THRnnxLc/edit?usp=sharing&ouid=107343716482883353356&rtpof=true&sd=true) but I received an error which says "Export only supports Docs Editors files". 
```
from langchain.document_loaders import GoogleDriveLoader
loader = GoogleDriveLoader(document_ids=["1SJhVh8rQE7gZN_iUnmHC9XF3THRnnxLc"], credentials_path='credentials.json', token_path='token.json')
docs = loader.load()
docs
```
HttpError: <HttpError 403 when requesting https://www.googleapis.com/drive/v3/files/1SJhVh8rQE7gZN_iUnmHC9XF3THRnnxLc/export?mimeType=text%2Fplain&alt=media returned "Export only supports Docs Editors files.". Details: "[{'message': 'Export only supports Docs Editors files.', 'domain': 'global', 'reason': 'fileNotExportable'}]">

**Comment by yyb0330** [comment]
It seems that docx files uploaded to Google Drive cannot be loaded, and I think only Google Docs format files are supported. The example file 'doggo wikipedia' is in docx format, so it should be converted to Google Docs format for use.

**Comment by Lensyi** [comment]
Seems to still be an issue

**Comment by DeanSingh** [comment]
GoogleDriveLoader as currently implemented can only download non google doc files.  Google doc files have to be exported due to Google's API.

This is blocking my Google Docs from being loaded via the lower.

**Comment by dmacoritto** [comment]
Did anyone figure out how to load google files ? I am still having this problem.

**Comment by DeanSingh** [comment]
Yeah I was able to get around it.  

I was able to call download_sheets directly along with just calling load().  Make sure the folder_id, document_ids, or file_ids are set.

**Comment by dmacoritto** [comment]
I did not really understand what you meant @DeanSingh, but I found a solution :-).

Actually, the GoogleDriveLoader has the `document_ids` and the `file_ids` parameter. The `document_ids` only accepts Google Docs whereas the `file_ids` accepts all the rest.

So based on the mimeType of your documents, you must ba able to split your documents either as Google docs or not. Then, just feed your documents ids to `file_ids` or `document_ids` based on their type.

It should give something like that:

```
GoogleDriveLoader(
        file_ids=non_google_file_to_load,
        document_ids=google_file_to_load,
        credentials_path=credentials_path / "credentials.json",
        token_path=credentials_path / "token.json",
        file_loader_cls=UnstructuredFileIOLoader,
        file_loader_kwargs={"mode": "single"}
)
```
